.eslintignore

```diff
@@ -1,7 +1,6 @@
 !.eslintrc.js
 build
 config
-contracts
```

Comment:
this line omitted linting in any directory with a `contracts` folder such as `src/logic/contracts`, but they still should be checked. I didn't find any reason to ignore this directory. Please let me know if I miss something